dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -44,15 +45,14 @@ protected String parseCommandText(String commandText) {
 
     public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
             throws SQLException {
-        String[] qualifiers = NameNormaliser.normaliseName(tableOrViewName)
-                .split("\\.");
+        String[] qualifiers = tableOrViewName.split("\\.");
         String qry = " select column_name, data_type, character_maximum_length "
                 + "	as direction from information_schema.columns where ";
 
         if (qualifiers.length == 2) {
-            qry += " lower(table_schema)=? and lower(table_name)=? ";
+            qry += " table_schema=? and table_name=? ";
```

Comment:
Are we making tables case-sensitive here? Are the other tests still working fine, will both below succeed:
```
|update|users|
|username=|name|
|adent2|arthur dent|
```

```
|UPDATE|USERS|
|USERNAME=|NAME|
|adent2|arthur dent|
```